File: solutions/beecrowd/1546/1546.cpp
#include <cstdint>
#include <iostream>

int main() {
    std::int16_t n, d, f;

    std::cin >> n;

    while (n--) {
        std::cin >> d;

        while (d--) {
            std::cin >> f;

            if (f == 1) {
                std::cout << "Rolien" << std::endl;
            } else if (f == 2) {
                std::cout << "Naej" << std::endl;
            } else if (f == 3) {
                std::cout << "Elehcim" << std::endl;
            } else {
                std::cout << "Odranoel" << std::endl;
            }
        }
    }

    return 0;
}

File: solutions/beecrowd/1546/1546.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

n = int(input())

for _ in range(n):
    d = int(input())

    for _ in range(d):
        f = int(input())

        if f == 1:
            output('Rolien\n')
        elif f == 2:
            output('Naej\n')
        elif f == 3:
            output('Elehcim\n')
        else:
            output('Odranoel\n')
